### PR TITLE
Message changes in entities

### PR DESCRIPTION
**Pull request by JoaoDiogoF**
changed some of the entities in the messages also commented them

### CODE REVIEW

File: src/main/java/ch/ntb/jass/common/proto/server_messages/ChooseGameModeMessage.java [hunk 1]
```diff
@@ -1,7 +1,14 @@
...
 import ch.ntb.jass.common.proto.ToPlayerMessage;
 
+/**
+ * This is a message from the server sent to a player
+ * Asks the player to choose a gamemode
+ * @param mode modes that he can choose
+ */
+
 public class ChooseGameModeMessage extends ToPlayerMessage {
-    public boolean canSchieben;
```
Comment (regonizer) [inline src/main/java/ch/ntb/jass/common/proto/server_messages/ChooseGameModeMessage.java]: this is an important node

File: src/main/java/ch/ntb/jass/common/proto/player_messages/ChosenGameModeMessage.java [hunk 2]
```diff
@@ -4,6 +4,12 @@
 import ch.ntb.jass.common.entities.GameMode;
 import ch.ntb.jass.common.proto.ToServerMessage;
 
+/**
+ * This Message is sent to the Server, it contains the information of the game mode
+ * @param mode contains the gamemode
+ * @param color contains the color the card with the chosen gamemode is "trumpf"
+ */
+
 public class ChosenGameModeMessage extends ToServerMessage {
```
Comment (mcalzaferri) [inline src/main/java/ch/ntb/jass/common/proto/player_messages/ChosenGameModeMessage.java]: Please use ChosenTrumpMessage instead. The Trump class already combines GameMode with a CardColor.

File: src/main/java/ch/ntb/jass/common/proto/server_info_messages/GameOverInfoMessage.java [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+import ch.ntb.jass.common.proto.ToPlayerMessage;
+
+/**
+ * This is a informationmessage sent to all players.
+ * It's the Information of the Game being over with the final Score
+ * @param score contains the score
+ */
+
+public class GameOverInfoMessage extends ToPlayerMessage {
+    public Score [] score;
```
Comment (mcalzaferri) [inline src/main/java/ch/ntb/jass/common/proto/server_info_messages/GameOverInfoMessage.java]: Is the use of an array necessary? Score could already contain the score of both teams.

**Comment by JoaoDiogoF** [comment]
changes are made @johannesneyer

**Pull request merged**